action-scheduler.php

```diff
@@ -26,21 +25,21 @@
  *
  */
 
-if ( ! function_exists( 'action_scheduler_register_1_dot_6_dot_0_dev' ) ) {
+if ( ! function_exists( 'action_scheduler_register_2_dot_0_dot_0_dev' ) ) {
 
 	if ( ! class_exists( 'ActionScheduler_Versions' ) ) {
 		require_once( 'classes/ActionScheduler_Versions.php' );
 		add_action( 'plugins_loaded', array( 'ActionScheduler_Versions', 'initialize_latest_version' ), 1, 0 );
 	}
 
-	add_action( 'plugins_loaded', 'action_scheduler_register_1_dot_6_dot_0_dev', 0, 0 );
+	add_action( 'plugins_loaded', 'action_scheduler_register_2_dot_0_dot_0_dev', 0, 0 );
 
-	function action_scheduler_register_1_dot_6_dot_0_dev() {
+	function action_scheduler_register_2_dot_0_dot_0_dev() {
 		$versions = ActionScheduler_Versions::instance();
-		$versions->register( '1.6.0', 'action_scheduler_initialize_1_dot_6_dot_0_dev' );
+		$versions->register( '1.6.0', 'action_scheduler_initialize_2_dot_0_dot_0_dev' );
```

Comment:
I think this line also needs `2.0.0` in place of `1.6.0`.

Reply:
Good catch. All the references to 1.6/1.6.0 in doc block and deprecation messages also need to be updated! I've fixed that up, rebased in the fixes and force pushed the branch.